spring-thymeleaf-2/src/main/resources/templates/mvcdata/email-request-parameters.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+	  xmlns:th="http://www.thymeleaf.org">
```

Comment:
Please double-check the indentation on the rest of the HTML files. It should use spaces instead of tabs. It should be a 2-space indent for continuations like these.